Configuration/Settings.yaml

```diff
@@ -1,3 +1,53 @@
+CodeQ:
+  LinkChecker:
+    # how many concurrent requests should the command controller perform
+    # If set too high, you most DOS your server
+    concurrency: 10
+    ignoreRobots: false
+    excludeStatusCodes: []
+
+    clientOptions:
+      cookies: true
+      connectionTimeout: 10
+      timeout: 10
+      allowRedirects: false
+      auth: []
+
+    notifications:
+      enabled: true
+      service: 'CodeQ\LinkChecker\Service\EmailService'
+      subject: 'Link checker results'
+      minimumStatusCode: 300
+      mail:
+        sender:
+          default:
+            name: 'CodeQ'
+            address: 'no-reply@example.com'
+        recipient:
+          default:
+            name: 'CodeQ'
+            address: 'test@example.com'
+        template:
+          package: 'CodeQ.LinkChecker'
+          file: 'NotificationMail'
+        attachment:
+          enableMailAttachment: true
+          csvHeader:
+            - 'Status code'
+            - 'URL'
+            - 'Origin URL'
+            - 'Domain'
+            - 'Checked at'
+        logging:
+          errors: log
+          success: log
+
+    urlsToCrawl:
```

Comment:
README and Settings do not match, please update the README.

Also please set it to emtpy by default, otherwise all these must be manually overriden.